torch/nn/modules/rnn.py

```diff
@@ -283,7 +285,7 @@ def __setstate__(self, d):
 
     @property
     def all_weights(self) -> List[Parameter]:
-        return [[getattr(self, weight) for weight in weights] for weights in self._all_weights]
+        return [[getattr(self, weight) for weight in weights] for weights in self._all_weights]  # type: ignore
```

Comment:
Is the problem that we're returning a List[List[Parameter]] and the signature is List[Parameter]?

Reply:
Yes. `weight` here has type `Parameter?